elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/filter/visitor/SplitFilterExpressionVisitorTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.filter.visitor;
+
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.Path;
+import com.yahoo.elide.core.filter.FilterPredicate;
+import com.yahoo.elide.core.filter.Operator;
+import com.yahoo.elide.core.filter.expression.AndFilterExpression;
+import com.yahoo.elide.core.filter.expression.FilterExpressionVisitor;
+import com.yahoo.elide.core.filter.expression.OrFilterExpression;
+import com.yahoo.elide.datastores.aggregation.example.PlayerStats;
+import com.yahoo.elide.parsers.expression.FilterExpressionNormalizationVisitor;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+
+public class SplitFilterExpressionVisitorTest {
+
+    private static final FilterExpressionNormalizationVisitor NORMALIZATION_VISITOR =
+            new FilterExpressionNormalizationVisitor();
+    private static final FilterPredicate WHERE_PREDICATE = new FilterPredicate(
```

Comment:
The RSQLDialect now has a simple method to convert from a RSQL expression to a FilterExpression.  That might be easier to construct queries - rather than do it this way.

Reply:
Hmm I like that!

I would like to keep this as static but RSQLDialect cannot be initialized without a entity dictionary which is not static. 

Well I think constructing in the current way looks very easy to me though :)